onyo/lib/onyo.py

```diff
@@ -38,57 +38,129 @@ def __init__(self, path: Union[Path, str], init: bool = False) -> None:
 
     @property
     def assets(self) -> set[Path]:  # pyre-ignore[11]
+        """
+        A `set` containing the `Path`s relative to repo.root to all assets of an
+        Onyo repository.
+
+        The cache will be consistent with the state of the repository as long as
+        only the `Repo`s public functions are used. Use of private functions
```

Comment:
"with `Repo.clear_caches()`" -> "See  `Repo.clear_caches()`."

I like referring them to the other docstring, because it will explain in more detail.